apps/client/src/components/challenges/form/ChallengesForm.tsx

```diff
@@ -51,41 +56,48 @@ export const ChallengesForm = (props: ChallengeFormProp) => {
 
     // other checks if any
     handleSubmit(values);
+    router.push('/challenges');
   };
-
+  const router = useRouter();
```

Comment:
Let's put declarations at the top of the function.

Reply:
Done